components/ee/ws-scheduler/pkg/scheduler/state_test.go

```diff
@@ -28,40 +28,95 @@ func TestState(t *testing.T) {
 		Expectation string
 	}{
 		{
-			Desc:        "no pods",
-			Nodes:       defaultNodeSet(),
-			Expectation: "- node1: used 0.000+0.000+0.000 of 10.000, avail 10.000 GiB\n- node2: used 0.000+0.000+0.000 of 10.000, avail 10.000 GiB\n- node3: used 0.000+0.000+0.000 of 10.000, avail 10.000 GiB",
+			Desc:  "no pods",
+			Nodes: defaultNodeSet(),
+			Expectation: `- node1:
```

Comment:
for future reference: in other places we use structs to express the expectation, which creates rather nice diffs when used with gocmp.

Reply:
Good point. Also, the parameters to `createWorkspacePod` could easily be `structs` as well... but I wanted to keep the diff minimal with this PR.